graylog2-web-interface/src/components/streams/StreamDetails/routing-destination/EditOutputButton.tsx

```diff
@@ -53,7 +53,7 @@ const EditOutputButton = ({ output, disabled = false, onUpdate, getTypeDefinitio
     <>
       <Button
         bsStyle="link"
-        disabled={!isPermitted(currentUser.permissions, 'stream:edit') || disabled}
+        disabled={!isPermitted(currentUser.permissions, 'outputs:edit') || disabled}
```

Comment:
Do we need both permissions here since this is in StreamDetails ?

Reply:
good question, IDK

Reply:
I think we do, because you need to have 'streams:edit' permission to get to edit stream related details.